src/v2/Apps/Settings/Routes/Purchases/Components/SettingsPurchasesRow.tsx

```diff
@@ -0,0 +1,351 @@
+import {
+  Box,
+  CheckCircleFillIcon,
+  Column,
+  EntityHeader,
+  Flex,
+  GridColumns,
+  HelpIcon,
+  Image,
+  PendingCircleIcon,
+  Separator,
+  Skeleton,
+  SkeletonBox,
+  SkeletonText,
+  Text,
+  XCircleIcon,
+} from "@artsy/palette"
+import { DateTime } from "luxon"
```

Comment:
We should enrich all the stitched date fields rather than having to import luxon for something like formatting.